Makefile

```diff
@@ -9,7 +9,13 @@ PROJ = google_containers
 
 PKG_SOURCES := $(shell find pkg -name '*.go')
 
-node-problem-detector: $(PKG_SOURCES) node_problem_detector.go
+vet:
+	go list ./... | grep -v "./vendor*" | xargs go vet
```

Comment:
This seems a little weird to me although it works for now.

go list is listing packages, while `grep -v "./vendor*"` seems to assume that it is listing files.
In the future, if there is a package called "abc/vendorefg", it will also be excluded.